README.md

```diff
@@ -5,6 +5,8 @@ Our greatest strength is our diversity. It is each of our right, to have and voi
 
 ## Empathy
 We treat all people with equal respect and dignity.
+1. Converse with Tact and Professionalism
+	1. Most people use chat mediums like Microsoft Teams and emails. It is robotic to a point that the person typing in words could easily be misinterpreted as rude and tactless. As an example, we should avoid telling people to "Google it", or "I can google it for you", because it’s rude and unnecessary. By saying “Google it,” we’re indulging in gratuitous meanness—a milder version of posting a sarcastic comment out of spite. We could also give the benefit of the doubt that the person asking the question must have already done that but came with inconclusive results. Regardless if the question seems unintelligent, we also don't know the answer to what is being asked or we don't have the time to answer, so rather than deflecting the obvious, we can simply phrase, "I don't have an answer for you and I think it's best to search online for possible answers".
```

Comment:
- There is a lot of specificity in this.
- We should focus on positive statements.